src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -103,7 +103,7 @@ protected void _terminate(TaskListener listener) throws IOException, Interrupted
 
       // If the instance is running, attempt to terminate it. This is an asynch call and we
       // return immediately, hoping for the best.
-      cloud.client.terminateInstanceWithStatus(cloud.projectId, zone, name, "RUNNING");
+      cloud.getClient().terminateInstanceWithStatus(cloud.projectId, zone, name, "RUNNING");
```

Comment:
I was just re-visiting https://issues.jenkins-ci.org/browse/JENKINS-52736 and noticed you thought this line should be changed. Is that still the case.

Reply:
It still should be changed..
I just changed call to client..